src/Ormolu/Printer/Meat/Common.hs

```diff
@@ -130,6 +132,18 @@ p_infixDefHelper isInfix indentArgs name args =
         breakpoint
         inciIf indentArgs $ sitcc (sep breakpoint sitcc args)
 
+-- | Print a Haddock.
+p_hsDoc ::
```

Comment:
It looks like `p_hsDocString` is not really useful on its own after this change. Could we remove it from the export list and possibly merge the two definitions so that we only have a longer definition of `p_hsDoc` directly?

Reply:
Removed `p_hsDocString` :+1: